<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PictureWidget</class>
 <widget class="QWidget" name="PictureWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>780</width>
    <height>240</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>PictureWidget</string>
  </property>
  <widget class="QPushButton" name="btnL">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>90</y>
     <width>25</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="btnR">
   <property name="geometry">
    <rect>
     <x>720</x>
     <y>90</y>
     <width>25</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>9</x>
     <y>9</y>
     <width>761</width>
     <height>225</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="pictureView" name="graphicsView"/>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <spacer name="horizontalSpacer_2">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="pictureButton" name="btnPic0" native="true"/>
      </item>
      <item>
       <widget class="pictureButton" name="btnPic1" native="true"/>
      </item>
      <item>
       <widget class="pictureButton" name="btnPic2" native="true"/>
      </item>
      <item>
       <widget class="pictureButton" name="btnPic3" native="true"/>
      </item>
      <item>
       <widget class="pictureButton" name="btnPic4" native="true"/>
      </item>
      <item>
       <widget class="pictureButton" name="btnPic5" native="true"/>
      </item>
      <item>
       <widget class="pictureButton" name="btnPic6" native="true"/>
      </item>
      <item>
       <widget class="pictureButton" name="btnPic7" native="true"/>
      </item>
      <item>
       <widget class="pictureButton" name="btnPic8" native="true"/>
      </item>
      <item>
       <widget class="pictureButton" name="btnPic9" native="true"/>
      </item>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <zorder>layoutWidget</zorder>
  <zorder>btnR</zorder>
  <zorder>btnL</zorder>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>pictureView</class>
   <extends>QGraphicsView</extends>
   <header location="global">pictureview.h</header>
  </customwidget>
  <customwidget>
   <class>pictureButton</class>
   <extends>QWidget</extends>
   <header location="global">picturebutton.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
